api/src/UserManager/LoginRegister/Test/fetchInput.test.ts

```diff
@@ -0,0 +1,72 @@
+import {expect} from "@jest/globals";
+import * as bcrypt from 'bcrypt';
+
+export interface ILoginForm {
```

Comment:
This should be a DTO not an Interface, I think we need to think and establish where we will have interfaces.

for me Interfaces should exist for Services only